### ISSUE TITLE
extractors: SQLite table layout change with OMNeT++ version 6

### ISSUE THREAD

**Issue by awillecke** [ibr-cm/ions#14]
With OMNeT++ version 6, the database layout of recorded results changed. The `params` are now `config` such that the old `runParams` table is now created like this:

```      
CREATE TABLE runConfig                                                                                                                                                                                                                                                                                                        
    (                                                                                                                                                                                                                                                                                                                         
        runId       INTEGER NOT NULL REFERENCES run(runId) ON DELETE CASCADE,                                                                                                                                                                                                                                                 
        configKey   TEXT NOT NULL,                                                                                                                                                                                                                                                                                            
        configValue TEXT NOT NULL,                                                                                                                                                                                                                                                                                            
        configOrder INTEGER NOT NULL                                                                                                                                                                                                                                                                                          
    );   
```

Unfortunately, the database file does not seem to store the OMNeT++ version number in the database file itself.

**Comment by hagau** [comment]
Hi, thanks for the report :slightly_smiling_face:. I've not yet worked with OMNeT++ v6, so I didn't even notice that change.

Since it's just a rename of the column names, this should be relatively easy to deal with. I'll look into this now.

**Comment by hagau** [comment]
I've just merged a solution (PR #16) to this into the main branch. As the PR states, it's a somewhat rough solution, but it works. I've tested it with the data produced by the ALOHA example shipped with OMNeT++.

If you still encounter problems with this, please feel free to reopen this issue again. And please open other issues related to OMNeT++ v6 should you encounter them :slightly_smiling_face:. I unfortunately won't be able to thoroughly test that version before the documentation is in a better state.

**Comment by awillecke** [comment]
Thank you for resolving this issue so quickly :)
